src/zcl/definition/cluster.ts

```diff
@@ -4571,6 +4571,8 @@ const Cluster: {
         attributes: {
             noise: {ID: 26112, type: DataType.data8},
             noise_detected: {ID: 26113, type: DataType.bitmap8},
+            NoiseDetectLevel: {ID: 26114, type: DataType.uint16},
+            NoiseAfterDetectDelay: {ID: 26115, type: DataType.uint16},
```

Comment:
Could you cleanup the naming a bit here by making it camel case? E.g. `noise_detected` -> `noiseDetected`, `NoiseDetectLevel` -> `noiseDetectLevel`

Reply:
Corrected, please check.